spec/code_objects/constants_spec.rb

```diff
@@ -66,7 +66,9 @@
   describe :BUILTIN_CLASSES do
     it "includes all base classes" do
       bad_names = []
-      YARD::CodeObjects::BUILTIN_CLASSES.each do |name|
+      builtin_classes = YARD::CodeObjects::BUILTIN_CLASSES
+      builtin_classes -= ["Bignum", "Fixnum"] if 0.class == Integer
```

Comment:
I'm going to decline this change and cherry-pick 3bfa1b2 only. In this case, silencing Ruby's deprecation is preferable (via `$VERBOSE = nil`)